### PR TITLE
Added fail message to test adding election policy

### PR DESCRIPTION
**Pull request by honza-kasik**

### CODE REVIEW

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/singleton/ElectionPolicyTestCase.java [hunk 1]
```diff
@@ -113,11 +114,12 @@ public void addSimpleElectionPolicyInUI() throws Exception {
...
                 .addResource(WizardWindowWithAdvancedSelectBoxOptions.class);
 
-        window.pick(SIMPLE)
+        window.pick(RANDOM)
                 .clickContinue()
                 .finish();
 
-        new ResourceVerifier(SINGLETON_POLICY_ADDRESS.and(ELECTION_POLICY, SIMPLE), client).verifyExists();
+        new ResourceVerifier(SINGLETON_POLICY_ADDRESS.and(ELECTION_POLICY, RANDOM), client)
+                .verifyExists("This fails because of https://issues.jboss.org/browse/HAL-1256");
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/singleton/ElectionPolicyTestCase.java]: Please add `probably` or something like that to the assert message.

**Review by zebrik — COMMENTED** [review]
Sorry for being such a pedant. Just one tiny optional complaint.

**Comment by honza-kasik** [comment]
Thanks! It's no problem! I edited the fail message and squashed.

**Comment by zebrik** [comment]
Thank you. Merging ...

**Pull request merged**